holoviews/plotting/mixins.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import, division, unicode_literals
```

Comment:
It would have been nice to add `mixins` in a separate PR to make it easier to see what is related to the new element. Too late now though!